script/BaseData.s.sol

```diff
@@ -255,6 +257,18 @@ contract BaseData is Script {
             LZ_EXECUTOR: 0x4208D6E27538189bB48E603D6123A94b8Abe0A0b,
             LZ_EID: 30329
         });
+        __chainIdToData[__chainIds.ink] = Data({
+            OFT_OWNER: 0x5848af047b56F7FCc9DFEAC2F535d4800069E9E1,
+            TOKEN_ADMIN: 0x5848af047b56F7FCc9DFEAC2F535d4800069E9E1,
+            PROXY_ADMIN: 0x5848af047b56F7FCc9DFEAC2F535d4800069E9E1,
+            LZ_ENDPOINT: 0xca29f3A6f966Cb2fc0dE625F8f325c0C46dbE958,
+            LZ_SEND_LIB: 0x76111DE813F83AAAdBD62773Bf41247634e2319a,
+            LZ_RECEIVE_LIB: 0x473132bb594caEF281c68718F4541f73FE14Dc89,
+            LZ_DVN: 0x174F2bA26f8ADeAfA82663bcf908288d5DbCa649,
+            NETHERMIND_DVN: 0xDd7B5E1dB4AaFd5C8EC3b764eFB8ed265Aa5445B,
+            LZ_EXECUTOR: 0x4208D6E27538189bB48E603D6123A94b8Abe0A0b,
```

Comment:
i can't locate this executor address
<img width="1087" alt="Screenshot 2025-04-09 at 3 00 50 AM" src="https://github.com/user-attachments/assets/2cff5c49-18f2-4dd4-be91-d6dc825e3101" />


There
s a different one listed here: https://docs.layerzero.network/v2/deployments/deployed-contracts

Reply:
good catch.  I've updated it and double checked the other addresses.